appveyor.yml

```diff
@@ -0,0 +1,14 @@
+init:
```

Comment:
Can you set the `DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true`, `DOTNET_CLI_TELEMETRY_OPTOUT=1` env variables in here?